cclib/parser/logfileparser.py

```diff
@@ -426,6 +426,21 @@ def append_attribute(self, name, value):
             self.set_attribute(name, [])
         getattr(self, name).append(value)
 
+    def _assign_coreelectrons_to_element(self, element, ncore, divide_by_count=False):
```

Comment:
Is there a clearer name for `divide_by_count`? Without looking how this function is used elsewhere, I think it's supposed to indicate whether `ncore` is a per-atom value instead of total # of core electrons across all atoms of given element. Perhaps `ncore_is_total_count` or `ncore_is_per_atom`?

Reply:
That was the original intent, yes.

Reply:
Improved this in a2b2e6458f5cf26de1743ab8077ff6fa9476904d

Reply:
Improved this in a2b2e6458f5cf26de1743ab8077ff6fa9476904d

Reply:
Improved this in a2b2e6458f5cf26de1743ab8077ff6fa9476904d

Reply:
Improved in 3a3f6edbb4dcda098f464408f63d7496006d4b6f

Reply:
Improved in 3a3f6edbb4dcda098f464408f63d7496006d4b6f